README.md

```diff
@@ -16,8 +16,45 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Prefer promises over callbacks](#prefer-promises-over-callbacks)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Prefer promises over callbacks
+### Why?
+Use promises instead of callbacks to keep code more readable and future proof when handling asynchronous requests.
```

Comment:
Promises can also be passed around so other modules can chain (`.then`) onto it, instead of complex callbacks inside callbacks (pyramid of doom) structures.